dagobah/core/dagobah.py

```diff
@@ -19,6 +19,7 @@ class Dagobah(object):
     instance, as well as top-level parameters such as the
     backend used for permanent storage.
     """
+    JIJ_DELIM = '%_|JIJ_DELIMITER|_%'
```

Comment:
very true. That's the kind of discussion I wanted to open up here- what level of risk vs readability vs customizability do we want to have?